extensions/mssql/src/objectManagement/interfaces.ts

```diff
@@ -443,6 +451,17 @@ export interface Database extends ObjectManagement.SqlObject {
 	azureServiceLevelObjective?: string;
 	azureEdition?: string;
 	azureMaxSize?: string;
+	autoCreateIncrementalStatistics: boolean;
```

Comment:
Out of curiosity why are some of these marked as optional with the ? and some aren't?

Reply:
The  boolean properties does have a value coming from SMO and for some properties we use nullable(including few booleans) so that we can consider them as unsupported when we get a null value to them.